source/working_remotely.rst

```diff
@@ -0,0 +1,60 @@
+################
+Working Remotely
+################
+
+Some nao robots are always turned on in the Robotics Lab for working remotely via the Kodos server.
+
+*****************************
+Connecting to Kodos using SSH
+*****************************
+
+First, your computer should be connected to Kodos using SSH.
+
+.. note::
+    For creating your account on Kodos, please :ref:`contact` us.
+
+Run the following command in a terminal of your computer to connect to Kodos.
+
+.. code-block:: bash
+
+   $ ssh -L 5903:localhost:5903 username@kodos.cse.unsw.edu.au
+
+***************************
+Running VNC Server in Kodos
+***************************
+
+Run :code:`vncserver -geometry <dim>` in Kodos such as
+
+.. code-block:: bash
+
+   $ vncserver -geometry 1920x893
+
+Run :code:`vncserver -list` in Kodos to find your X Display NO.
```

Comment:
the X display number and tcp port also appear when you run vncserver

Reply:
New changes were made to this file.